server/pulp/server/db/model/__init__.py

```diff
@@ -51,3 +52,51 @@ class RepositoryContentUnit(Document):
                     'fields': ['unit_id']
                 }
             ]}
+
+
+class Repository(Document):
+    """
+    Defines schema for a pulp repository in the `repos` collection.
+
+    :ivar repo_id: unique across all repos
+    :type repo_id: mongoengine.StringField
+    :ivar display_name: user-readable name of the repository
+    :type display_name: mongoengine.StringField
+    :ivar description: free form text provided by the user to describe the repo
+    :type description: mongoengine.StringField
+    :ivar notes: arbitrary key-value pairs programmatically describing the repo;
+                 these are intended as a way to describe the repo usage or
+                 organizational purposes and should not vary depending on the
+                 actual content of the repo
+    :type notes: mongoengine.DictField
+    :ivar content_unit_counts: key-value pairs of number of units associated with this repo.
+                               This is different than the number of associations, since a
+                               unit may be associated multiple times.
+    :type content_unit_counts: mongoengine.DictField
+    :ivar metadata: arbitrary data that describes the contents of the repo;
```

Comment:
I copied this over from the Repo "model". I guess this was removed at some point. Good catch.